### PR TITLE
refs #3455: Fix conversion

### LINKED ISSUE #3455: Check of conversions
Continuing 
https://github.com/modelica/ModelicaStandardLibrary/issues/3181#issuecomment-549259166

Old classes missing (note, I had https://github.com/modelica/ModelicaStandardLibrary/pull/3453 checked out, but I don't think that matters):
- [x] Modelica.Magnetic.FundamentalWave.BaseClasses.Machine
- [x] Modelica.Magnetic.FundamentalWave.BasicMachines.InductionMachines.{IM_SlipRing,IM_SquirrelCage}
- [x] Modelica.Magnetic.FundamentalWave.BasicMachines.SynchronousMachines.{SM_ElectricalExcited,SM_PermanentMagnet,SM_ReluctanceRotor}
- [x] Modelica_Synchronous.RealSignals.TimeBasedSources.Sin (for conversion of f->freqHz) (should be Sine) See #3456 for fix.

I also tried to check that all public elements in MSL are converted.

That seems to be the case for classes in MSL (excluding Examples, UserGuide, and partial ones).

For components conversions are missing for some "internal" variables, e.g., vbc in PNP due to https://github.com/modelica/ModelicaStandardLibrary/pull/3178
but for parameters and graphical ones only the following seems to lack conversions:
- [x] Modelica.Electrical.Machines.Utilities.SwitchYD idealCommutingSwitch
- [x] Modelica.Electrical.Spice3.Internal.MOS2 vp
- [x] Modelica.Magnetic.QuasiStatic.FundamentalWave.Utilities.SwitchYD idealCommutingSwitch
- [x] Modelica.Mechanics.MultiBody.Parts.Body z_a_start
- [x] Modelica.Fluid.Machines.BaseClasses.PartialPump show_NPSHa
I'm not sure if any of them are relevant.

(The above is excluding classes that are partially converted.)

### PR DESCRIPTION
**Pull request by beutlich**
Remove conversion from new class.

Is there anything more to fix? Was not that obvious for me.

Fix #3455.

### CODE REVIEW

**Comment by beutlich** [comment]
What's the state on this PR and https://github.com/beutlich/ModelicaStandardLibrary/pull/4?

**Comment by christiankral** [comment]
I kindly ask @HansOlsson to have look at the PR https://github.com/beutlich/ModelicaStandardLibrary/pull/4 again and double check if the updated proposal is formally OK by now. From my personal point of view this PR is OK. Due to https://github.com/beutlich/ModelicaStandardLibrary/pull/4#issuecomment-594038025 I can unfortunately not fully check in Dymola 2020 if my proposal is correct or not.

**Comment by HansOlsson** [comment]
I have now merged the PR from @christiankral and checked the result, and all of the reported issues are resolved.

**Comment by dietmarw** [comment]
@HansOlsson not quite. The last two merges were not necessary and messed the PR up a bit. I reseted those and force pushed it again into the PR.

**Comment by HansOlsson** [comment]
> @HansOlsson not quite. The last two merges were not necessary and messed the PR up a bit. I reseted those and force pushed it again into the PR.

I see. I guess I should have created the branch using:
`git checkout -b fix-conversion beutlich/fix-conversion`

**Comment by dietmarw** [comment]
Yes and then 
`git merge christiankral/fix-conversion fix-conversion`
What I have done is basically force pushing `christiankral/fix-conversion` into this PR's branch `beutlich/fix-conversion`. Just to have a nicer/linear history:

image

Without merge commits.

**Comment by dietmarw** [comment]
@christiankral can we get your approval?

**Comment by HansOlsson** [comment]
I have now added corrections for three additional issues, will add description in original ticket.

File: Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos [hunk 1]
```diff
@@ -1846,7 +1846,7 @@ convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.VoltageS
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.VoltageSensor","arg_y","arg_v")
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "y", "i")
 convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "abs_y", "abs_i")
-convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "arg_y", "arg_i")
+convertElement("Modelica.Electrical.QuasiStationary.SinglePhase.Sensors.CurrentSensor", "arg_y", "arg_v")
```
Comment (beutlich) [inline Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos]: That probably is a bug that should be reverted.

Comment (HansOlsson) [reply Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos]: Ah, you mean a bug in the CurrentSensor, right? Yes, that is likely correct when I spend two seconds thinking of it.

Comment (beutlich) [reply Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos]: Yes, was fixed by #3488 and should be reverted here.

Comment (HansOlsson) [reply Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos]: Ok, done!

Comment (beutlich) [reply Modelica/Resources/Scripts/Conversion/ConvertModelica_from_3.2.3_to_4.0.0.mos]: I simply removed these two back and forth commits to get a clean history when merging.

**Pull request merged**